notebooks/useful_notebooks/01_find_and_release.ipynb

```diff
@@ -100,7 +100,28 @@
     "if len(all_exps_on_sets) != len(all_exps):\n",
     "    print('Number of experiments is not same as experiments associated with sets')\n",
     "    print('# of exps: {}. # of exps on sets: {}'.format(len(all_exps), len(all_exps_on_sets)))\n",
-    "        \n",
+    "\n",
+    "hela_exps = []  # a list of exp['uuid'] with biosource from HeLa individual\n",
+    "hela_exps_unsure = []\n",
+    "for exp_type in exp_names:\n",
+    "    for exp in store[exp_type]:\n",
+    "        biosource_is_hela = 0\n",
+    "        for bs in exp['biosample']['biosource']:\n",
+    "            if bs.get('individual') and bs['individual'].get('display_title') == '4DNINEL8T2GK':  # the HeLa individual\n",
+    "                biosource_is_hela += 1\n",
+    "        if biosource_is_hela == len(exp['biosample']['biosource']):  # all biosources are HeLa: exp is HeLa\n",
+    "            hela_exps.append(exp['uuid'])\n",
+    "        elif biosource_is_hela > 0:  # some but not all biosources are HeLa: unsure\n",
+    "            hela_exps_unsure.append(exp['uuid'])\n",
+    "if hela_exps_unsure and change_level >= 9:\n",
+    "    release_hela = input('Experiments with multiple Biosources found, some of which are HeLa. ' +\n",
+    "                         'Sequence files associated with these experiments will be restricted. ' +\n",
+    "                         'Do you want to release them, instead? (yes/no)')\n",
+    "    if release_hela == 'no':\n",
+    "        hela_exps.extend(hela_exps_unsure)\n",
+    "    elif release_hela != 'yes':\n",
+    "        raise KeyError('Invalid response')\n",
```

Comment:
Why KeyError? it seems like a ValueError would be more appropriate.

Reply:
```suggestion
    "        raise ValueError('Invalid response')\n",
```